CMakeLists.txt

```diff
@@ -838,8 +838,8 @@ if(CC_USE_AUDIO)
         ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/OpenSLHelper.h
         ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/PcmAudioPlayer.cpp
         ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/PcmAudioPlayer.h
-        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/PcmAudioService.cpp
-        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/PcmAudioService.h
+        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/openharmony/PcmAudioService.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/openharmony/PcmAudioService.h
```

Comment:
Why remove android/PcmAudioService.h/.cpp, will it break Android platform support?

Reply:
It seems only openharmony platform use this CMakeLists.txt.  Right?

Reply:
right,  only openharmony platform use this CMakeLists.txt.